app/javascript/controllers/access_rights_controller.js

```diff
@@ -2,23 +2,31 @@ import { Controller } from '@hotwired/stimulus'
 
 // This handles editing the access rights for a DRO
 export default class extends Controller {
-  static targets = ["view", "download", "location", "cdl", "downloadRow", "locationRow", "cdlRow"]
```

Comment:
Changes to `app/javascript/**/*.js` are to bring the code in line with the linter.